Corona/Service/JHUWebDataService.swift

```diff
@@ -17,6 +17,8 @@ public class JHUWebDataService: DataService {
 	private static var reportsURL: URL {
 		URL(string: "https://services1.arcgis.com/0MSEUqKaxRlEPj5g/arcgis/rest/services/ncov_cases/FeatureServer/1/query?f=json&where=Confirmed%20%3E%200&returnGeometry=false&spatialRel=esriSpatialRelIntersects&outFields=*&orderByFields=Confirmed%20desc%2CCountry_Region%20asc%2CProvince_State%20asc&resultOffset=0&resultRecordCount=500&cacheHint=false&rnd=\(Int.random())")!
 	}
+	private static var germanyReportsURL: URL = URL(string: "https://services7.arcgis.com/mOBPykOjAyBO2ZKk/arcgis/rest/services/Coronaf%C3%A4lle_in_den_Bundesl%C3%A4ndern/FeatureServer/0/query?f=json&where=1%3D1&returnGeometry=false&spatialRel=esriSpatialRelIntersects&outFields=*&orderByFields=Fallzahl%20desc&resultOffset=0&resultRecordCount=25&cacheHint=true")!
```

Comment:
This class is only for the data from JHU. Since you're adding a data source, it makes more sense to create a new class for that. Something like `<source_name>DataService`. And keep this class for JHU only.

Reply:
I guess your concern is not repeating code. So, I think we can move the shared methods to a common base class. Something like `BaseDataSource`

Reply:
I wouldn't mind if you take it from here ;-) Thanks.

Reply:
But yes, I was thinking about doing a BaseDataSource. I guess it will be faster if you do it on your side. You only need to copy my files and it should look like this at the end: https://twitter.com/StuFFmc/status/1251908016636669953

Reply:
No problem 😆